pkg/backends/datadog/datadog.go

```diff
@@ -30,7 +30,7 @@ const (
 	// BackendName is the name of this backend.
 	BackendName                  = "datadog"
 	dogstatsdVersion             = "5.6.3"
-	dogstatsdUserAgent           = "python-requests/2.6.0 CPython/2.7.10"
+	defaultDogstatsdUserAgent    = "gostatsd"
```

Comment:
nit: defaultUserAgent is a better name

Reply:
Ahh, yeah.  I initially didn't change the value, only made it configurable.  Will update this.